python/doc_generator.py

```diff
@@ -1,18 +1,34 @@
+import argparse
 import os
-from typing import Dict
+from typing import Dict, List, Any
+
+import pyrasgo
 
 from python import utils, markdown as md
 
 DOCS_DIR = utils.get_root_dir() / 'docs'
 
+DATASET_PREVIEW_TEXT = """Pull a source Dataset and preview it:"""
+
+DATASET_PREVIEW_CODE = """ds = rasgo.get.dataset(id)\nprint(ds.preview())"""
```

Comment:
As mentioned in another comment, I think it would be best if there is a `1 to 1` exact mapping of the code executed to create the output example df and the one shown in the docs, ideally using the same string to execute the python code with `eval(python_code)`.

What do you think about this?